src/main/java/edu/unc/lib/boxc/migration/cdm/services/FieldAssessmentTemplateService.java

```diff
@@ -0,0 +1,141 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.poi.xssf.usermodel.XSSFColor;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.apache.poi.xssf.usermodel.XSSFFont;
+import org.apache.poi.xssf.usermodel.XSSFCellStyle;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.SheetConditionalFormatting;
+import org.apache.poi.ss.usermodel.ConditionalFormattingRule;
+import org.apache.poi.ss.usermodel.FillPatternType;
+import org.apache.poi.ss.usermodel.IndexedColors;
+import org.apache.poi.ss.usermodel.PatternFormatting;
+import org.apache.poi.ss.util.CellRangeAddress;
+
+import edu.unc.lib.boxc.migration.cdm.model.CdmFieldInfo;
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+
+/**
+ * @author krwong
+ */
+public class FieldAssessmentTemplateService {
+    private CdmFieldService cdmFieldService;
+
+    public void generate(MigrationProject project) throws IOException {
+        // Get a list of all the fields
+        cdmFieldService.validateFieldsFile(project);
+        CdmFieldInfo fieldInfo = cdmFieldService.loadFieldsFromProject(project);
+        List<CdmFieldInfo.CdmFieldEntry> fields = fieldInfo.getFields().stream()
+                .filter(f -> !f.getSkipExport())
+                .collect(Collectors.toList());
+
+        // Create workbook and sheet
+        XSSFWorkbook workbook = new XSSFWorkbook();
+        Sheet sheet = workbook.createSheet("Template");
+
+        // Style spreadsheet
+        XSSFFont defaultFont = workbook.createFont();
+        defaultFont.setFontHeightInPoints((short) 11);
+        defaultFont.setFontName("Calibri");
+        defaultFont.setColor(IndexedColors.BLACK.getIndex());
+        defaultFont.setBold(false);
+        defaultFont.setItalic(false);
+
+        XSSFFont fontBold = workbook.createFont();
+        fontBold.setFontHeightInPoints((short) 11);
+        fontBold.setFontName("Calibri");
+        fontBold.setColor(IndexedColors.WHITE.getIndex());
+        fontBold.setBold(true);
+        fontBold.setItalic(false);
+
+        XSSFCellStyle defaultStyle = workbook.createCellStyle();
+        defaultStyle.setFont(defaultFont);
+
+        XSSFCellStyle headerStyle = workbook.createCellStyle();
+        headerStyle.setFillForegroundColor(new XSSFColor(new java.awt.Color(0, 102, 204), null));
+        headerStyle.setFillPattern(FillPatternType.SOLID_FOREGROUND);
+        headerStyle.setFont(fontBold);
+
+        SheetConditionalFormatting sheetCF = sheet.getSheetConditionalFormatting();
+        ConditionalFormattingRule rule1 = sheetCF.createConditionalFormattingRule("=MOD(ROW(),2)=0");
```

Comment:
that's pretty cool, didn't know you could do that

Reply:
isn't it? I had fun playing with the formatting :)